src/openms/thirdparty/evergreen/src/Engine/MessagePasser.hpp

```diff
@@ -154,7 +154,7 @@ class MessagePasser {
     return _number_edges_with_messages_received == number_edges() || (_number_edges_with_messages_received+1 == number_edges() && !_edge_received[edge_index]);
   }
 
-  virtual bool ready_to_send_message_ab_initio(unsigned long edge_index) const {
+  virtual bool ready_to_send_message_ab_initio(unsigned long /*edge_index*/) const {
```

Comment:
This is syntax I am not familiar with. Out of curiosity how does it work?

Reply:
This is a multiline comment in C++.
Any text between /* and */ will be ignored by the compiler.
You can also use it to comment out a a block of code (hence the name), without touching every line as you would when using the single line comment `//`